lib/plugins/physics.js

```diff
@@ -220,6 +223,48 @@ function inject (bot) {
     return vel.x * vel.x + vel.z * vel.z
   }
 
+  function liquidPush(pos){
+    let flow = liquidFlow(pos);
+    let push = flow.clone().scaled(physics.liquidPush);
+    push.y = push.y/physics.gravity;
+    console.log(flow);
+    return push;
+  }
+
+  function liquidFlow(pos) {
+    let flow = new Vec3(0,0,0);
+    let base = bot.blockAt(pos).metadata;
+    let id = bot.blockAt(pos).type;
+    if(base > 7) {
+      base = 0;
+      flow.set(0,-1,0);
+    }
+    const depth = level => {
+      if(level > 7) return 0;
+      return level; 
+    }
+    let Blocks = {};
+    Blocks['x+'] = bot.blockAt(pos.offset(1,0,0));
+    Blocks['x-'] = bot.blockAt(pos.offset(-1,0,0));
+    Blocks['z+'] = bot.blockAt(pos.offset(0,0,1));
+    Blocks['z-'] = bot.blockAt(pos.offset(0,0,-1));
+    let waterBlocks = {}
+    for(let i = 0; i < Object.keys(Blocks).length; i++){
+      key = Object.keys(Blocks)[i];
```

Comment:
use for with `Object.entries()`

Reply:
I will fix it and run standard --fix

Reply:
but don't I need the key so I can add entries to the waterBlocks object?